sql/analyzer/pushdown.go

```diff
@@ -258,7 +258,7 @@ func pushdownFiltersToAboveTable(
 		if err != nil {
 			return nil, transform.SameTree, err
 		}
-		filters.markFiltersHandled(handled...)
+		filters.markFiltersHandled(tableFilters...)
```

Comment:
(minor) might be worth a comment mentioning why you pass in `tablefilters`instead of `handled`. It wouldn't be obvious to me reading the code and might prevent someone from reverting this change.

Reply:
Fixed.